cfgen/cfgen

```diff
@@ -3,7 +3,7 @@
 import abc
 import argparse
 import ast
-from enum import Enum, auto
+from enum import Enum, auto, IntEnum
```

Comment:
Test comment

Reply:
Sorry, didn't get the question, could you please elaborate?

Reply:
Please ignore.
It was only to test if commenting can be done for this PR.